services/spar/test-integration/Util.hs

```diff
@@ -356,25 +346,30 @@ ping :: (Request -> Request) -> Http ()
 ping req = void . get $ req . path "/i/status" . expect2xx
 
 
--- | Create a cloned 'NewIdP' corresponding to the mock idp from the config and suitable for
+makeIssuer :: MonadIO m => m Issuer
+makeIssuer = do
+  uuid <- liftIO UUID.nextRandom
+  either (liftIO . throwIO . ErrorCall . show)
+         (pure . Issuer)
+         (SAML.parseURI' ("https://issuer/_" <> UUID.toText uuid))
```

Comment:
should this be `issuer` or `issuer.net`?

Reply:
right, it should be consistent with the earlier issuer.net.  will fix!

(beyond that it doesn't matter.  it has to be a valid URL, but it is never used as anything but an identifier.)